<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1599</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/power.png);
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>200</number>
    </property>
   </widget>
   <widget class="QPushButton" name="downArrowButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/down.png);
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upArrowButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/up.png);
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/check.png);
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="connectionBottom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>51</width>
      <height>121</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|3|&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|2|&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|1|&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="connectionMiddle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>51</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|6|&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|5|&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|4|&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="connectionTop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|8|&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#888a85;&quot;&gt;|7|&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alphaButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/alpha.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deltaButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/delta.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="betaButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/beta.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="thetaButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/theta.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="group20Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/20.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="group45Button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/45.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="groupUserButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/user.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/CES.png);
	background-color: white;
	border-radius: 40;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="batteryLevel1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="batteryLevel2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="batteryLevel3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>-21</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>-20</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTextBrowser" name="log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>256</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CONSOLE</string>
     </property>
    </widget>
    <widget class="QSlider" name="connectionSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>410</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CONNECTION</string>
     </property>
    </widget>
    <widget class="QSlider" name="batterySlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>460</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>440</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Level</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>USER STUFF</string>
     </property>
     <widget class="QPushButton" name="addUserButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="userNameInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>104</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="userIntensityInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>104</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
     <widget class="QComboBox" name="nameComboBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>72</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="printHistoryButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Print history</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="historySpinBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="playReplayButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Play replay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addFakeRecordingButton">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>140</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>add fake recording</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>powerButton</zorder>
   <zorder>downArrowButton</zorder>
   <zorder>upArrowButton</zorder>
   <zorder>checkButton</zorder>
   <zorder>connectionBottom</zorder>
   <zorder>connectionMiddle</zorder>
   <zorder>connectionTop</zorder>
   <zorder>alphaButton</zorder>
   <zorder>deltaButton</zorder>
   <zorder>betaButton</zorder>
   <zorder>thetaButton</zorder>
   <zorder>group20Button</zorder>
   <zorder>group45Button</zorder>
   <zorder>groupUserButton</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>batteryLevel1</zorder>
   <zorder>batteryLevel2</zorder>
   <zorder>batteryLevel3</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1599</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
